tutorials/gm-world.md

```diff
@@ -68,7 +68,7 @@ If you get errors of `gmd` not found, you may need to add the `go/bin` directory
 :::
 
 ```bash
-curl -sSL https://rollkit.dev/install-gm-rollup.sh | sh
+cd $HOME && curl -sSL https://rollkit.dev/install-gm-rollup.sh | sh
```

Comment:
do we need to do this @yarikbratashchuk ? I think it looks much cleaner as:
```
curl -sSL https://rollkit.dev/install-gm-rollup.sh | sh
```

but would be ideal if it is not actually downloaded using something similar to:

```bash
bash -c "$(curl -sL https://docs.celestia.org/celestia-app.sh)"
```